openhtf/core/phase_executor.py

```diff
@@ -199,9 +199,10 @@ def execute_phase(self, phase):
     while not self._stopping.is_set():
       outcome = self._execute_phase_once(phase)
```

Comment:
make this return `PhaseOutcome(openhtf.PhaseResult.CONTINUE)` and undo everything else